airbyte-integrations/connectors/source-mixpanel/unit_tests/unit_test.py

```diff
@@ -1,12 +1,63 @@
 #
 # Copyright (c) 2021 Airbyte, Inc., all rights reserved.
 #
-
+import logging
 from datetime import date, timedelta
 
 from airbyte_cdk.sources.streams.http.auth import NoAuth
 from source_mixpanel.source import Annotations
+from source_mixpanel.source import SourceMixpanel
+
+logger = logging.getLogger("test_client")
+
+def test_check_connection_api_secret_ok(requests_mock):
+
+    config = {}
+    config["api_secret"] = "testApiSecret"
```

Comment:
You can do `config = {"api_secret": "testApiSecret"}`.

Reply:
done